linkatos/message.py

```diff
@@ -13,3 +15,16 @@ def contains_a_link(message):
         answer = answer.group(0).strip()
 
     return answer
+
+
+yes_re = re.compile("(Yes|YES|yes)")
+
+def contains_a_yes(message):
```

Comment:
This function makes more sense if it returns a boolean instead of a string or `None`. You actually want to know if there is a `yes` or no, you don't want to process the string further right?